datafusion/core/tests/fuzz_cases/sort_fuzz.rs

```diff
@@ -136,9 +136,9 @@ impl SortTest {
                     .sort_spill_reservation_bytes,
             );
 
-            let runtime_config = RuntimeConfig::new()
-                .with_memory_pool(Arc::new(GreedyMemoryPool::new(pool_size)));
-            let runtime = Arc::new(RuntimeEnv::new(runtime_config).unwrap());
+            let runtime_env = RuntimeConfig::new()
```

Comment:
not sure why it is env, config is more appropriate imho

Reply:
That's because a newly introduced method `build` returns env. Please check the rest of this expression in diff below.

Reply:
I agree with @comphead  this now looks a bit strange -- I filed https://github.com/apache/datafusion/issues/12156 with an idea of how to improve it